code/modules/mob/living/carbon/human/monkey/monkey.dm

```diff
@@ -34,6 +34,18 @@
 
 GLOBAL_DATUM(the_one_and_only_punpun, /mob/living/carbon/human/species/monkey/punpun)
 
+/obj/item/clothing/under/punpun
+	name = "fancy uniform"
+	desc = "It looks like it was tailored for a monkey."
+	icon = 'icons/obj/clothing/under/civilian.dmi'
+	worn_icon = MONKEY_UNIFORM_FILE
+	icon_state = "punpun"
+	can_adjust = FALSE
+	female_sprite_flags = NO_FEMALE_UNIFORM
+	species_exception = list(/datum/species/monkey)
+	bodytypes_restricted = BODYTYPE_MONKEY
```

Comment:
restricted sounds like youre allowing everyone except monkeys to wear it. why not just whitelist